playwright/tests/login-tests.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { test, expect, Page } from "@playwright/test";
+
+interface LoginFields {
+  username: string;
+  password: string;
+}
+
```

Comment:
Here you can use the interface as object type:
const credentials: LoginFields = {
username: "student",
password: "Password123"
}